block/sync.go

```diff
@@ -75,7 +70,14 @@ func (m *Manager) SettlementSyncLoop(ctx context.Context) error {
 
 				err = m.ApplyBatchFromSL(settlementBatch.Batch)
 				if err != nil {
-					return fmt.Errorf("process next DA batch. err:%w", err)
+					m.logger.Error("Apply batch from SL", "err", err)
```

Comment:
the problem I see with logging and breaking here is that as a node operator you'll never know if you have an issue. 

imagine u use a wrong celestia rpc or the rpc is down and you need to change it.
in that case you simply break and the node operator has no idea that he needs to change it. that's why we prefer to emit health issues.

the rpc retry loop should be long enough to indicate there is an rpc problem and not operator should do something about it (the node operator doesn't look at the logs but have alerts to `health` status)

Reply:
ok makes sense
I'll revert

Reply:
i think that  what we need to  do here is only log  and break in case the error is a da error (ErrRetrieval or ErrBlobNotfound), in the other cases return error as usual. This way if da fails the node will not stop and retry in the next state update but fail in case applyblock actually fails and is not da related.